includes/settings/functions.php

```diff
@@ -503,6 +503,39 @@ function admin_welcome_notice() {
 	<?php
 }
 
+function admin_no_payment_pointer_notice() {
+	$screen = get_current_screen();
+	if ( ! $screen ) {
+		return;
+	}
+
+	if ( $screen->id !== 'toplevel_page_coil_settings' ) {
+		return;
+	}
+
+	$current_user       = wp_get_current_user();
+	$payment_pointer_id = Admin\get_global_settings( 'coil_payment_pointer_id' );
+
+	if ( $payment_pointer_id) {
+		return;
+	}
+
+	$active_tab = isset( $_GET['tab'] ) ? sanitize_text_field( $_GET['tab'] ) : '';
+
+	if ( $active_tab !== 'global_settings' ) {
+		return;
+	}
+	?>
+
+	<div style="display: none;" class="notice coil-no-payment-pointer-notice">
```

Comment:
Suggestion: is it worth only showing this nag if the user has the ability to edit the payment pointer field? e.g. someone with limited access to wp-admin would see this nag on the screen, but be unable to directly rectify it.